README.md

```diff
@@ -371,6 +371,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [rnoweb](https://github.com/bamonroe/tree-sitter-rnoweb) (maintained by @bamonroe)
 - [x] [robot](https://github.com/Hubro/tree-sitter-robot) (maintained by @Hubro)
 - [x] [ron](https://github.com/amaanq/tree-sitter-ron) (maintained by @amaanq)
+- [x] [roc](https://github.com/nat-418/tree-sitter-roc) (maintained by @nat-418)
```

Comment:
Position in the list seems wrong

Reply:
bot will take care of it, but might as well do it right.

Reply:
(Also lockfile)